src/zarr_benchmarks/plotting_functions.py

```diff
@@ -6,50 +6,47 @@
 from matplotlib import pyplot as plt
 
 
+def get_limits_custom(x_min: int, x_max: int, max_range: int) -> tuple[float, float]:
+    central_value = (x_min + x_max) / 2
+    x_lim_min = central_value - max_range / 2 - round(max_range, 1) / 10
+    x_lim_max = central_value + max_range / 2 + round(max_range, 1) / 10
+    return x_lim_min, x_lim_max
+
+
 def set_axes_limits(graph: sns.relplot, data: pd.DataFrame, plot_name: str) -> None:
```

Comment:
```suggestion
def set_axes_limits(graph: sns.relplot, data: pd.DataFrame) -> None:
```
`plot_name` is no longer used in this function